src/main/java/com/dm/dmservicesandbox/domain/UserToken.java

```diff
@@ -0,0 +1,31 @@
+package com.dm.dmservicesandbox.domain;
+import jakarta.persistence.*;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@NoArgsConstructor
+@Getter
+@Setter
+@Table(name = "user")
+public class UserToken

```

Comment:
What is this class for?

Reply:
This is used to update DB for email verification. Save the temp token and expire date in DB before verification, and clear them after verification. Also change user status to "verifying" before use click the link; "verified" after verification is successful.